sdk_container/src/third_party/coreos-overlay/coreos-base/afterburn/files/0004-cargo-codegen-units-release-profile.patch

```diff
@@ -0,0 +1,28 @@
+From bd688cfb2c39fea83bcde40a6b96abb94459bed4 Mon Sep 17 00:00:00 2001
+Message-Id: <bd688cfb2c39fea83bcde40a6b96abb94459bed4.1701857811.git.dpark@linux.microsoft.com>
+From: Dongsu Park <dpark@linux.microsoft.com>
+Date: Wed, 6 Dec 2023 11:03:15 +0100
+Subject: [PATCH] Cargo: set 1 codegen-unit reduce binary size
+
+Add `codegen-unit = 1` in case of release profile, to reduce binary
+size of afterburn included in production images.
+---
+ Cargo.toml | 2 ++
+ 1 file changed, 2 insertions(+)
+
+diff --git a/Cargo.toml b/Cargo.toml
+index da74d909..2b12776d 100644
+--- a/Cargo.toml
++++ b/Cargo.toml
+@@ -32,6 +32,8 @@ path = "src/main.rs"
+ [profile.release]
+ # We assume we're being delivered via e.g. RPM which supports split debuginfo
+ debug = true
```

Comment:
I think we can remove the `debug = true` because we throw the result away. Setting `strip = true`  would also make it clearer what the end result on the image will look like.

Reply:
Done.

Yes, both adding `strip = true` and removing `debug = true` help reduce the binary size.
After being installed, it looks anyway nearly the same as before.